src/js/engagement_view/src/components/engagementView/sidebar/tables/lensTable/lensTableStyles.tsx

```diff
@@ -56,7 +56,12 @@ export const useStyles = makeStyles((theme) =>
             width: "95%",
         },
         tableRow: {
-            background: theme.palette.background.default,
+            "&.Mui-selected": {
+                backgroundColor: "#2B3648",
+                "& > .MuiTableCell-root": {
+                    color: "#1976D2",
+                },
+            },
```

Comment:
It's a bit odd to me that these are the only bare hex-colors in this file. Is there any way we can associate them with the Theme palette? Would that make sense to do or are they unrelated?

Reply:
Moved to theme, now referenced in pallete